src/View/ViewableData.php

```diff
@@ -537,7 +537,7 @@ protected function objCacheClear()
      * @param array $arguments
      * @param bool $cache Cache this object
      * @param string $cacheName a custom cache name
-     * @return Object|DBField
+     * @return object|DBField
```

Comment:
It's probably the case that this used to mean the old SS3 `Object` which later became `SS_Object` which is effectively `ViewableData` now from a typing perspective.

That said, the way this is implemented, it really can return any object, so I'm going to treat this as though it was just a capital letter away from being correct initially.